src/Client.php

```
@@ -58,22 +58,20 @@ public function disableLimit()
      * @param array $payload
      *
      * @return bool
-     * @throws \Exception
      */
     public function send(array $payload)
     {
-        $isLimitEnabled = $this->isLimitEnabled;
-        $isExceedLimit = $this->getNotify()->isExceedLimit();
-
-        if ($isLimitEnabled && $isExceedLimit) {
+        if ($this->isLimitEnabled && $this->getNotify()->isExceedLimit()) {
             return false;
         }
 
-        if ($isLimitEnabled) {
-            $this->getNotify()->increment();
+        try {
+            $this->getSlack()->send($payload);
+        } catch (\Exception $exception) {
```

Actually, I think it's better to throw an Exception. but not common Exception. SlackNotifyException like that.
and also put source Exception information.